tasks/sbd_watchdog.yml

```
@@ -0,0 +1,46 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Configure SBD watchdog
+  block:
+    - name: SBD - Configure and unload watchdog kernel modules from blocklist
+      block:
+        - name: "SBD - Configure watchdog kernel module blocklist"
+          ansible.builtin.lineinfile:
```

```suggestion
          lineinfile:
```
please don't use FQCN for ansible built-ins (I know, the Ansible guidelines say to do it, but not for system roles)

---

done